pkg/authorization/api/types.go

```diff
@@ -20,18 +20,53 @@ const (
 	VerbAll     = "*"
 )
 
+const (
+	ResourceGroupPrefix         = "resourcegroup"
+	BuildGroupName              = ResourceGroupPrefix + ":builds"
+	DeploymentGroupName         = ResourceGroupPrefix + ":deployments"
+	ImageGroupName              = ResourceGroupPrefix + ":images"
+	OAuthGroupName              = ResourceGroupPrefix + ":oauth"
+	UserGroupName               = ResourceGroupPrefix + ":users"
+	PolicyOwnerGroupName        = ResourceGroupPrefix + ":policy"
+	PermissionGrantingGroupName = ResourceGroupPrefix + ":granter"
+	OpenshiftExposedGroupName   = ResourceGroupPrefix + ":exposedopenshift"
```

Comment:
is there a clear line between exposed and private openshift and kube resources? maybe I just haven't worked with the resources enough, but I wouldn't have been able to tell you which things should go in each group prior to looking on Friday (and even now, I've forgotten and don't have any idea which things are in each group).